src/main/java/Akinator.java

```diff
@@ -0,0 +1,210 @@
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import java.util.NoSuchElementException;
+import java.util.Scanner;
+import java.util.concurrent.TimeUnit;
+
+
+public class Akinator {
+    public static WebDriver driver;
+
+    public static void main(String[] args) throws InterruptedException, NoSuchElementException {
+        initiateGame();
+        answerQuestions();
+        confirmYourHero();
+
+        endGame();
+    }
+
+    //    Різниця між implicitlyWait & sleep. Розібратися що коли краще використовувати для економії часу на проходження
+    public static void initiateGame() {
+        System.setProperty("webdriver.chrome.driver", "src/test/resources/chromedriver.exe");
+
+        driver = new ChromeDriver();
+        driver.manage().window().maximize();
+        secWait(10);
+        driver.get("https://ru.akinator.com");
+        clickCss("a[href='/game']");
+    }
+
+    private static void secWait(int sec) {
+        driver.manage().timeouts().implicitlyWait(sec, TimeUnit.SECONDS);
+    }
+
+    public static void endGame() {
+        driver.quit();
+    }
+
+    public static void showFiveOptions() {
+
+        String question = driver.findElement(By.xpath("//p[@class='question-text']")).getText();
+        System.out.println("Question: " + question);
+        System.out.println("Варианты ответа:");
+        String first = textCss("a[id='a_yes']");
```

Comment:
Could you please rename `textCss` and add a verb to that describes method's intent?